scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/impl/ProvenanceTest.java

```diff
@@ -358,6 +359,35 @@ public void testAdaptedOpWithDependencies() {
 	 */
 	@Test
 	public void testSimplificationRecovery() {
+		// Get the Op
+		Computers.Arity1<ObjectArray<Number>, ObjectArray<Number>> c = ops //
+				.op("test" + ".provenanceComputer") //
+				.arity1() //
+				.inType(new Nil<ObjectArray<Number>>() {}) //
+				.outType(new Nil<ObjectArray<Number>>() {}) //
+				.computer();
+		// Get the signature from the Op
+		String signature = ops.history().signatureOf(c);
+		// Generate the Op from the signature and an Op type
+		Nil<Computers.Arity1<ObjectArray<Number>, ObjectArray<Number>>> special = new Nil<>() {};
+		Computers.Arity1<ObjectArray<Number>, ObjectArray<Number>> fromString = ops.opFromSignature(
+				signature, special);
+		// Assert Op similarity
+		Assertions.assertTrue(wrappedOpEquality(c, fromString));
+		// Assert Op functionality similarity
+		ObjectArray<Number> in = new ObjectArray<>(new Number[] { 1, 2, 3 });
+		ObjectArray<Number> actual = new ObjectArray<>(new Number[] { 0, 0, 0 });
+		fromString.compute(in, actual);
+		ObjectArray<Number> expected = new ObjectArray<>(new Number[] { 1., 2., 3. });
+		Assertions.assertEquals(expected, actual);
+	}
+
+	/**
+	 * Tests the ability of {@link OpEnvironment#opFromSignature(String, Nil)} to
+	 * generate an Op that has been adapted and simplified.
+	 */
+	@Test
+	public void testFocusedRecovery() {
 		// Get the Op
 		Computers.Arity1<Integer[], Integer[]> c = ops //
 			.op("test" + ".provenanceComputer") //
```

Comment:
Why is this string split with the `+` operator here?

Reply:
Fixed!